toolbox/util/githubClient.go

```diff
@@ -126,7 +126,11 @@ func (g GithubClient) Remote(repo string) string {
 
 // CreatePullRequest within :repo from :branch to :baseBranch
 func (g GithubClient) CreatePullRequest(
-	title, body, branch, baseBranch, repo string) (*github.PullRequest, error) {
+	title, body, releaseNote, branch, baseBranch, repo string) (*github.PullRequest, error) {
```

Comment:
I don't think changing the function signature is a good idea since this function has been used at multiple places and for each occurrence you need to update the params. A better approach would be function overloading, which defines another CreatePullRequest() with an additional param `releaseNote`. From line 130, you could then call the overloaded CreatePullRequest() with `releaseNote = none`

Reply:
It is not clear what empty string means when reading the code. I hope my comment makes sense.

Reply:
1. Do you know someone are using this function outside test-infra? I think I changed every usages in test-infra. For us, most of auto PR needs to have "release-note" part. Actually now in istio, all PR has to have that part so I believe it's a good time to upgrade this function. 
2. Right, I am adding comment to make it clearer.